core/src/main/scala/com/mesosphere/usi/core/CachedPendingLaunch.scala

```diff
@@ -10,19 +15,28 @@ case class CachedPendingLaunch(pendingLaunch: Set[PodId] = Set.empty) {
     *
     * @param podIds
     */
-  def update(specs: SpecState, state: SchedulerState, podIds: Set[PodId]): CachedPendingLaunch = {
+  def update(specs: SpecState, state: SchedulerState, podIds: Set[PodId]): UpdateResult = {
     var newPendingLaunch = pendingLaunch
+    var newToBeLaunched = Set.empty[PodId]
```

Comment:
What's the difference between `newPendingLaunch` and `newToBeLaunched`?

Reply:
hmm yeah, I struggled with the names here. Do you have any suggestions? Basically:
- pending launch are those who are still waiting to be launched (it's a superset of all)
- new to be launched are instances waiting to be launched that were added in this frame